Gordon360/Controllers/RecIMControllers/LeagueController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+
+namespace Gordon360.Controllers.RecIMControllers
+{
+    [Route("api/recim")]
+    public class LeagueController : GordonControllerBase
```

Comment:
By convention, controllers should have a plural name. This is too match the plural route:
```suggestion
    public class LeaguesController : GordonControllerBase
```

Reply:
Got it, just updated